server/users/views.py

```diff
@@ -0,0 +1,17 @@
+from django.shortcuts import render
+from rest_framework.authtoken.views import ObtainAuthToken
+from rest_framework.authtoken.models import Token
+from rest_framework.response import Response
+
+class CustomAuthToken(ObtainAuthToken):
+
+    def post(self, request, *args, **kwargs):
+        serializer = self.serializer_class(data=request.data,
```

Comment:
I think you can just create your serializer and use email as the field instead of username

Reply:
[This](https://github.com/encode/django-rest-framework/blob/master/rest_framework/authtoken/views.py) is the serializer they used, we can write our own @TaslimOwolarafe